wiki/People/Users_with_unique_titles/id.md

```diff
@@ -184,17 +187,17 @@ Storyboard yang menunjukkan bakat disertakan juga cerita yang luar biasa dan men
 - ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231)
 - ![][flag_CN] [yf_bmp](https://osu.ppy.sh/users/1243669)
 
-### Terkait dengan gameplay
+### Terkait dengan permainan
 
-Para pemain ini mendapatkan gelar mereka melalui pencapaian tergila dalam game ini atau telah menyelesaikan tantangan tertentu.
+Pemain-pemain berikut memperoleh gelarnya masing-masing atas keberhasilan mereka dalam menorehkan rekor-rekor yang luar biasa atau menyelesaikan tantangan-tantangan tertentu yang terkait dengan permainan.
 
 | Pengguna | Gelar | Pencapaian |
 | :-- | :-- | :-- |
-| ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | Mencetak NoMod FC pertama di [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |
-| ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Mencetak FC pada semua beatmap berstatus Ranked pada game mode osu!taiko hingga 3 November 2019 |
+| ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | Menorehkan FC NoMod pertama pada beatmap [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |
```

Comment:
```suggestion
| ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | Mendapatkan FC No-Mod pertama pada beatmap [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |
```

Reply:
o you don't spell `NoMod` as `No-Mod` orz. what is currently in the page is already correct :(

![image](https://user-images.githubusercontent.com/36564236/120115046-28eba680-c1ac-11eb-9070-da15f1cabd64.png)